classifier.go

```diff
@@ -10,7 +10,7 @@ import (
 // The scores must order the candidates so as the highest score be the most probably language of the content. The candidates is
 // a map which can be used to assign weights to languages dynamically.
 type Classifier interface {
-	Classify(content []byte, candidates map[string]float64) map[string]float64
+	Classify(content []byte, candidates map[string]float64) (scores map[float64]string)
```

Comment:
A map with a float64 key looks really awkward. Is really a map from float to strings what you want? Or just an slice with language and weight pairs or similar?